### ISSUE TITLE
关于验证集的问题

### ISSUE THREAD

**Issue by byhbyhyoyo** [Coder-Yu/SELFRec#54]
SimGCL论文的原文中有这样一段话：“We split the datasets into three parts (training set, validation set, and test set) with a ratio of 7:1:2.”
  但是我看数据集中并没有提供验证集，只有训练集和测试集，这是为什么呢？

**Comment by Coder-Yu** [comment]
得到最终结果的时候把训练集和验证集merge了，这个主要是为了跟SGL论文里面的结果比较。他们采用了这种方式
